golden_response/staging/get_stat_ranking/california.json

```diff
@@ -0,0 +1,594 @@
+{
+  "payload":  {
+    "Count_Person":  {
+      "rankAll":  {
+        "info":  [
+          {
+            "value":  10039107,
```

Comment:
I assume the "rank": 0 is not shown because it's default proto value. Might be better to show it (with some proto field option tuning)

Reply:
I took a look into it... We can add EmitUnpopulated: true to the marshaller option, but that would only apply to unit tests. The rank: 0 is stored in the cache, but there's no way to differentiate it in go, or to set an option at the field level. I'm not sure if that's what we want (other unset fields will then show up, in this case relatedPlaces: [], etc).

I could go back and fix this in prophet, starting the rank values at 1? That might make more sense.

Reply:
I see, right, starting with 1 in prophet sounds good.